### PR TITLE
Flaky Test detector: skip slow tests

### PR DESCRIPTION
**Pull request by marcoandredinis**
Some tests can now be skipped when running the Flaky Tests Detector.
It also adds two tests that are slow enough to hit the timeout: TestCompletenessReset and TestCompletenessInit

Demo:
image

### CODE REVIEW

File: lib/cache/cache_test.go [hunk 1]
```diff
@@ -602,7 +602,7 @@ func TestCompletenessInit(t *testing.T) {
 
 	// put lots of CAs in the backend
 	for i := 0; i < caCount; i++ {
-		ca := suite.NewTestCA(types.UserCA, fmt.Sprintf("%d.example.com", i))
+		ca := suite.NewTestCA(types.UserCA, fmt.Sprintf("%d.example.org", i))
```
Comment (ryanclark) [inline lib/cache/cache_test.go:605]: Is this change needed?

Comment (marcoandredinis) [reply lib/cache/cache_test.go]: Not really
It was only meant for the screenshot, but should do no harm
Reverting anyway :+1:

**Review by camscale — APPROVED** [review]

**Review by ryanclark — APPROVED** [review]
